src/editor/Collaboration.js

```diff
@@ -0,0 +1,15 @@
+define(function (require, exports, module) {
+    "use strict";
+
+    var togetherjs = require("togetherjs");
+
+    function Collaboration() {
+        this.togetherjs = togetherjs;
+        var self = this;
+        window.setTimeout(function() {
```

Comment:
Using a timeout is a bad idea, let's just use existing events, or add a new one.  What are you trying to achieve exactly?

* There are a bunch of handlers for various startup events here: https://github.com/mozilla/brackets/blob/master/src/utils/AppInit.js#L130-L147
* https://github.com/mozilla/brackets/blob/master/src/extensions/default/bramble/lib/RemoteEvents.js#L237-L273 is when we are 100% loaded and running as a usable editor, and we could easily emit an internal event here